### PR TITLE
Feature/footer logo position

### PR DESCRIPTION
**Pull request by rs-mgb**

### CODE REVIEW

File: src/es/components/controllers/events/Events.js [hunk 1]
```diff
@@ -97,6 +97,7 @@ export default class Events extends Shadow() {
       ...options
     }, ...args)
 
+    this.separator = '💩'
```
Comment (Weedshaker) [inline src/es/components/controllers/events/Events.js:100]: 😂

File: src/es/components/controllers/events/Events.js [hunk 2]
```diff
@@ -211,6 +212,13 @@ export default class Events extends Shadow() {
         }))
       })
     }
+
+     // set Monday as first day in Datepickr
```
Comment (Weedshaker) [inline src/es/components/controllers/events/Events.js:216]: use the "this.getDateOptionEventsListener'-Object to return this in the object. A controller should never use a querySelector.

Comment (Weedshaker) [reply src/es/components/controllers/events/Events.js:216]: i fixed it here... so you should be able to delete L215-222

Comment (Weedshaker) [reply src/es/components/controllers/events/Events.js:216]: https://github.com/mits-gossau/web-components-toolbox/commit/bdebe1974569d1500b0117af08cb0b8061640ec2#diff-28617f8fdaae0e1e22dbbf887a04b79c8920e9146528481d947504e7e538356fR201

**Review by Weedshaker — CHANGES_REQUESTED** [review]
Please fix the querySelector stuff from the Events.js controller.

File: src/es/components/controllers/events/Events.js (hunk 2 above)
Comment (rs-mgb) [reply src/es/components/controllers/events/Events.js]: Removed

**Review by edmgb — APPROVED** [review]

**Pull request merged**